mailpoet/webpack.config.js

```diff
@@ -107,7 +108,8 @@ const baseConfig = {
       ),
     },
   },
-  plugins: PRODUCTION_ENV ? [] : [new ForkTsCheckerWebpackPlugin()],
+  plugins:
+    PRODUCTION_ENV || CIRCLECI_ENV ? [] : [new ForkTsCheckerWebpackPlugin()],
```

Comment:
An update — I just run into this and realized this shouldn't be needed at all as [we run the builds with the production env on CI anyway](https://github.com/mailpoet/mailpoet/blob/dcdd1c8cdb9773672b015c48fb5a814119bf6dbb/.circleci/config.yml#L151).

Reply:
I'm a bit confused about this. Would that affect the JS tests too? For example this pipeline: https://app.circleci.com/pipelines/github/mailpoet/mailpoet/15351/workflows/8a8d463a-53db-498b-9251-f14eb7cf1f55/jobs/260417